Meridian59/Data/Models/AdminData/AdminData.cs

```diff
@@ -0,0 +1,153 @@
+﻿using System;
+using System.ComponentModel;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Meridian59.Common.Interfaces;
+using Meridian59.Data.Lists;
+using Meridian59.Protocol.Events;
+using Meridian59.Protocol.GameMessages;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminData : INotifyPropertyChanged, IClearable
+    {
+        //Handles the objects we're watching in the AdminUI
+        private BaseList<AdminObject> TrackedObjects { get; set; }
+        public void TrackAdminObject(AdminObject adminObject)
+        {
+            TrackedObjects.Add(adminObject);
+            OnTrackedAdminObjectAdded(new AddTrackedAdminObjectEventHandlerArgs(this, adminObject));
+        }
+        public void UntrackAdminObject(AdminObject adminObject)
```

Comment:
A bit like with the comments above...

Instead of providing a "TrackAdminObject()" and "UntrackAdminObject()" method, observe the trackedObjects collection using its "ListChanged" event which you subscribe to in the constructor here.

Then from outside simply do:
>> Data.AdminData.TrackedObjects.Add(...)
>> Data.AdminData.TrackedObjects.Remove(..)

This way you're subscribed handlers will be called.
It's like hooking in, rather than forcing someone to call 'your' methods.